MovieApp/PresentationLayer/Views/Components/MovieInfoCell/MovieInfoCell+Design.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import UIKit
+
+extension MovieInfoCell: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    func createViews() {
+        contentContainer = UIView()
+        contentView.addSubview(contentContainer)
+
+        poster = UIImageView()
+        contentContainer.addSubview(poster)
+
+        infoContainer = UIView()
+        contentContainer.addSubview(infoContainer)
+
+        nameLabel = UILabel()
+        infoContainer.addSubview(nameLabel)
+
+        aboutLabel = UILabel()
+        infoContainer.addSubview(aboutLabel)
+    }
+
+    func styleViews() {
+        styleCell()
+        styleContents()
+    }
+
+    private func styleCell() {
+        contentContainer.layer.cornerRadius = cornerRadius
+        contentContainer.backgroundColor = .white
+        contentContainer.clipsToBounds = true
+    }
+
+    private func styleContents() {
+        nameLabel.attributedText = NSMutableAttributedString(
+            string: "",
+            attributes: [.font: UIFont.systemFont(ofSize: 16, weight: .bold)]
+        )
+
+        aboutLabel.attributedText = NSMutableAttributedString(
+            string: "",
+            attributes: [.font: UIFont.systemFont(ofSize: 14, weight: .regular)]
+        )
+        aboutLabel.numberOfLines = 0
+    }
+
+    func defineLayoutForViews() {
+        contentContainer.snp.makeConstraints {
+            $0.top.leading.trailing.equalToSuperview()
+            $0.bottom.equalToSuperview().inset(12)
```

Comment:
this inset is not needed... if you are using it to create space between cells, you should do that via collection view flow layout